### ISSUE TITLE
bcml doesnt open.

### ISSUE THREAD

**Issue by gu1lt** [NiceneNerd/BCML#489]
i dont know if its because im on windows 11 but the exe for bcml doesnt open

**Comment by GingerAvalanche** [comment]
Try the console command `py -m bcml` and then using the settings menu to make shortcuts.

Need to figure out why the executable no longer works. Leaving this open as a reminder.

**Comment by giever** [comment]
Using the console command `py -m bcml` works to launch it, and I can create shortcuts with the settings menu once in, but those shortcuts also don't launch BCML. In the console, `bcml-debug` also works by the way, but regular `bcml` doesn't.

**Comment by GingerAvalanche** [comment]
Thanks. That helps narrow down the problem. Open it with `py -m bcml` until I can get out a fix.

**Comment by GingerAvalanche** [comment]
For anyone whose shortcuts don't work, right click on the shortcut and select Properties. In the `Target` field, replace `pythonw.exe` with `python.exe` (just delete the `w`) and it should work until I can figure out why Python isn't building the exe correctly.

**Comment by tlf30** [comment]
Same issue, but I cannot launch via command line. 
```
c:\Users\Trevor\AppData\Local\Programs\Python\Python310\Scripts>py -m bcml
Traceback (most recent call last):
  File "C:\Users\Trevor\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\Trevor\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\Trevor\AppData\Local\Programs\Python\Python310\lib\site-packages\bcml\__main__.py", line 159, in <module>
    main()
  File "C:\Users\Trevor\AppData\Local\Programs\Python\Python310\lib\site-packages\bcml\__main__.py", line 93, in main
    import cefpython3
  File "C:\Users\Trevor\AppData\Local\Programs\Python\Python310\lib\site-packages\cefpython3\__init__.py", line 74, in <module>
    raise Exception("Python version not supported: " + sys.version)
Exception: Python version not supported: 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
```

Looks like a python versioning issue...

**Comment by svioletg** [comment]
I'm getting the same issue as above, but with 3.10.8. I updated BCML after following the prompt it gave me when I opened it, and it stopped launching after that - so I installed it through pip, and now I get `Exception: Python version not supported: 3.10.8 (tags/v3.10.8:aaaf517, Oct 11 2022, 16:50:30) [MSC v.1933 64 bit (AMD64)]`.

**Comment by tlf30** [comment]
I used pyenv to install it into python 3.9.13, that worked fine. Looks like an incompatibly with python 3.10

**Comment by GingerAvalanche** [comment]
That's a different error, so I opened a different issue for it.

The error this issue concerns is fixed in 3.10.4
